[OSDOCS#17058]: DC-2: Image Mirroring and oc mirror Tooling-CQA-1 (#115599)
openshift/openshift-docs: modules/oc-mirror-oci-format.adoc

apiVersion: mirror.openshift.io/v1alpha2
storageConfig:
  local:
    path: /home/user/metadata
mirror:
  platform:
    channels:
    - name: stable-{product-version}
      type: ocp
    graph: false
  operators:
  - catalog: oci:///home/user/oc-mirror/my-oci-catalog
    targetCatalog: my-namespace/redhat-operator-index
    packages:
    - name: aws-load-balancer-operator
  - catalog: registry.redhat.io/redhat/redhat-operator-index:v{product-version}
    packages:
    - name: rhacs-operator
  additionalImages:
  - name: registry.redhat.io/ubi9/ubi:latest
----
+
where:

`storageConfig.local.path`:: Sets the back-end location to save the image set metadata to. This location can be a registry or local directory. It is required to specify `storageConfig` values.
`mirror.platform.channels.name`:: Optionally, includes an {product-title} release to mirror from `registry.redhat.io`.

> `mirror.operators.catalog`:: Specifies the Operator catalog to mirror, either as an OCI path on disk starting with `oci://` or as a registry path to the catalog image.

Reviewer: I would keep the old text here, and add a second sentence that includes the text from callout 5:

```suggestion
`mirror.operators.catalog`:: Specifies the absolute path to the location of the OCI catalog on disk. The path must start with `oci://` when using the OCI feature. Optionally, you can specify additional Operator catalogs to pull from a registry.
```

apiVersion: mirror.openshift.io/v1alpha2
storageConfig:
  local:
    path: /home/user/metadata
mirror:
  platform:
    channels:
    - name: stable-{product-version}
      type: ocp
    graph: false
  operators:
  - catalog: oci:///home/user/oc-mirror/my-oci-catalog
    targetCatalog: my-namespace/redhat-operator-index
    packages:
    - name: aws-load-balancer-operator
  - catalog: registry.redhat.io/redhat/redhat-operator-index:v{product-version}
    packages:
    - name: rhacs-operator
  additionalImages:
  - name: registry.redhat.io/ubi9/ubi:latest
----
+
where:

> `storageConfig.path`:: Sets the back-end location to save the image set metadata to. This location can be a registry or local directory. It is required to specify `storageConfig` values.

Reviewer 2: ```suggestion
`storageConfig.local.path`:: Sets the back-end location to save the image set metadata to. This location can be a registry or local directory. It is required to specify `storageConfig` values.
```

`mirror.platform.channels.name`:: Optionally, includes an {product-title} release to mirror from `registry.redhat.io`.
`mirror.operators.catalog`:: Specifies the absolute path to the location of the OCI catalog on disk. The path must start with `oci://` when using the OCI feature.
`mirror.operators.targetCatalog`:: Optionally, specifies an alternative namespace and name to mirror the catalog as.

> `mirror.operators.catalog`:: Optionally, specifies additional Operator catalogs to pull from a registry.

Reviewer 2: this is duplication. Please revise the description to reflect only 1 instance of this field.

+
[source,terminal]
----
$ oc mirror --config=./imageset-config.yaml \
  docker://registry.example:5000 --v1
----
+
where:
+

`imageset-config.yaml`:: Passes in the image set configuration file. This procedure assumes that it is named `imageset-config.yaml`.

> `docker://`:: Specifies the registry to mirror the content to. The registry must start with `docker://`. If you specify a top-level namespace for the mirror registry, you must also use this same namespace on subsequent executions.

Reviewer 2: isn't `registry.example:5000` the user input?

> $ oc mirror --config=./imageset-config.yaml \ <1>

Reviewer 3: ```suggestion
$ oc mirror --config=./imageset-config.yaml \
```

>   docker://registry.example:5000 --v1             <2>

Reviewer 3: ```suggestion
  docker://registry.example:5000 --v1
```
